File: DOTNET/BaiduMapTest/GeoCoderTest.cs
using BaiduMap.Request;
using BaiduMap.Request.Models;
using System;
using System.Collections.Generic;
using System.Text;
using Xunit;
using Shouldly;

namespace BaiduMapTest
{
    public class GeoCoderTest
    {
        [Fact]
        public void Test()
        {
            var baiduClient = ClientCreator.Create();
            var model = new GeoCoderModel()
            {
                Address = "杭州市西湖区浙大路38号",
            };
            var req = new GeoCoderRequest(model);
            var resp = baiduClient.Execute(req);
            resp.Status.ShouldBe(0);
        }
    }
}

File: JAVA/baidumap-webapi/src/test/java/wang/seamas/baidumap/GeoCoderTest.java
package wang.seamas.baidumap;

import com.sun.tools.javac.jvm.Code;
import org.junit.Assert;
import org.junit.Test;
import wang.seamas.baidumap.request.GeoCoderRequest;
import wang.seamas.baidumap.request.model.GeoCoderModel;
import wang.seamas.baidumap.response.GeoCoderResponse;

/**
 * @author: Seamas Wang
 * @description:
 * @date: Created in 11:19 2018/8/6
 */
public class GeoCoderTest {

    @Test
    public void test() {
        BaiduApiClient client = ClientCreator.Create();

        GeoCoderModel model = new GeoCoderModel();
        model.setAddress("杭州市西湖区浙大路38号");

        GeoCoderRequest request = new GeoCoderRequest(model);
        GeoCoderResponse response = client.execute(request);
        Assert.assertEquals(0, response.getStatus());
    }
}
